### PR TITLE
Fix pacman version with php-fpm

### PR DESCRIPTION
**Pull request by Meklad**
- Add to PhpFpm->getPhpVersion() new condition to check package manager to skip the version for some bistros, and return the version or null depend on the bistro.

- The issue reference to this pull request: https://github.com/cpriego/valet-linux/issues/425

### CODE REVIEW

File: .phpunit.result.cache
Comment (jmarcher) [inline .phpunit.result.cache:1]: Please remove this file

File: cli/Valet/PhpFpm.php [hunk 1]
```diff
@@ -534,12 +534,22 @@ public function fpmConfigPath($phpVersion = null)
...
     }
 
-    public function getPhpVersion()
+    /**
+     * This method check package manager to skip the version for some distros,
+     * and return the version or null depend on the distro.
+     *
+     * @return string|null
+     */
+    public function getPhpVersion(): string|null
```
Comment (jmarcher) [inline cli/Valet/PhpFpm.php:543]: Valet should work in PHP <8 (php >7.0) the return type is not required.

**Comment by arcanna** [comment]
Can this fix be merged in ASAP? Unable to install valet on Manjaro/Arch for 2 weeks now due to this php8.2-fpm error

File: cli/Valet/PhpFpm.php
Comment (coffeeneer) [inline cli/Valet/PhpFpm.php:1]: Please add the using Pacman on line 10:

`use Valet\PackageManagers\Pacman;`

**Comment by jmarcher** [comment]
> Can this fix be kindly merged in ASAP? Unable to install valet on Manjaro/Arch for 2 weeks now due to this php8.2-fpm error. Please advise if there is an alternate workaround. Reverted to using an older version for now.

This PR can not be merged as it is

**Pull request closed without merging**